api/src/pcapi/core/mails/__init__.py

```diff
@@ -9,6 +10,9 @@
 from .backends.logger import LoggerBackend
 
 
+VALID_LOOKING_EMAIL_ADDRESS = re.compile(".+@.+\..+")
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Je me demande si on a pas déjà ce type de regex dans le code, par exemple à la création de compte.
J'ai pas trouvé en 2 min de ctrl+F :disappointed:

Reply:
Edit: je vois des choses qui semblent faire le taf ici: src/pcapi/utils/email.py

Reply:
Edit: je vois des choses qui semblent faire le taf ici: src/pcapi/utils/email.py